packages/frontend/src/api/document_rpc.test.ts

```diff
@@ -1,10 +1,11 @@
 import { type DocHandle, Repo, isValidDocumentId } from "@automerge/automerge-repo";
 import { BrowserWebSocketClientAdapter } from "@automerge/automerge-repo-network-websocket";
 import { type FirebaseOptions, initializeApp } from "firebase/app";
-import { createUserWithEmailAndPassword, deleteUser, getAuth, signOut } from "firebase/auth";
+import { deleteUser, getAuth, signOut } from "firebase/auth";
 import * as uuid from "uuid";
 import { assert, afterAll, describe, test } from "vitest";
 
+import { initTestUserAuth } from "../util/test_util.ts";
 import { createRpcClient } from "./rpc.ts";
 import { unwrap, unwrapErr } from "./test_util.ts";
```

Comment:
I can't think of a good reason to have two different modules for testing utils at this stage, so I'd suggest moving the content of `api/test_util.ts` to your new `util/test_util.ts` and then deleting the former file.

Reply:
I:
- Edited the PR to match conventions
- consolidated test utils in `util/test_util.ts`
- Removed the empty user profile test

Do you have a convention for who marks these kinds of conversations as resolved?

Reply:
Thanks for the changes! No fixed convention. If you find it helpful to resolve them yourself to keep track of what you've done, that's fine; otherwise, I'll do it.